omnibus/package-scripts/chef/postinst

```diff
@@ -99,10 +99,22 @@ ln -sf $INSTALLER_DIR/bin/chef-client $PREFIX/bin || error_exit "Cannot link che
 chown -Rh 0:0 $INSTALLER_DIR
 
 # make the base structure for chef to run
-mkdir -p /etc/chef/client.d/
-mkdir -p /etc/chef/accepted_licenses
-mkdir -p /etc/chef/trusted_certs
-mkdir -p /etc/chef/ohai/plugins
+# the sample client.rb is only written out of no chef config dir exists yet
+if ! [ -d $CONFIG_DIR ]; then
```

Comment:
`test` instead of `[` maybe, but i think that might be sol9 compat

Reply:
We have a bunch of the [ tests in this file already so that should work.